type.py

```diff
@@ -107,7 +107,6 @@ class Experiment:
     run_name: str  # Get's appended as a prefix before the pipeline name
     train: bool  # Weather the run should do training
     dataset_category: DatasetSplit
-    dataloader: "DataLoader"
```

Comment:
so I used this extensively when we're declaring different Experiments for different datasets, while keeping the Pipelines the same. Is it possible to retain this ability somehow?